tools.py

```diff
@@ -160,24 +160,35 @@ def du(input_path):
 	)
 
 
-def csv2json(input_path, gz, group, reset_duration):
+def csv2json(input_path, gz, group, reset_begin_end, csv_sep):
+	""" Convert cvs transcripts file to .csv.json transcripts file. Each line in `input_path` file must have format:
+		"audio_path,transcription,begin,end\n"
+		csv_sep could be "comma", representing ",", or "tab", representing "\t".
+	"""
```

Comment:
This file contains csv2json tool refactoring. This tool is widely used in the project, in the process of creating the datasets. Each dataset uses .csv and .csv.json file format conventions.  I introduced a csv_sep parameter, which lets you control, which csv separator you want to use (tab by default).